test-helpers/index.ts

```diff
@@ -100,13 +100,13 @@ export async function setupApplication(
 				primary: {
 					client: 'sqlite3',
 					connection: {
-						filename: '${join(fs.basePath, 'primary.sqlite3')}',
+						filename: '${join(fs.basePath, 'primary.sqlite3').split(sep).join(posix.sep)}',
```

Comment:
A little confused on why we doing this?

Reply:
If we don't convert the path to POSIX format on windows I have a config/database.ts file that looks like this after running `setupApplication` :
```ts
  connections: {
    primary: {
      client: 'sqlite3',
      connection: {
        filename: 'C:\Users\Julien\\adonis\adonis-auth\test-helpers\__app\primary.sqlite3',
      },
    },
  }
````

So it doesn't work coz some characters are escaped and therefore impossible to launch the tests